### PR TITLE
Triage 2024 07 30

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

File: triage/2024-07-30.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+| All ❌✅ (primary)                 | 0.1%  | [-0.3%, 0.7%]  | 8     |
+
+
+* primary regressions are to doc-full for html5ever, stm32f4, libc, and typenum
+* those are presumably due to PR [#126247](https://github.com/rust-lang/rust/issues/126247); pnkfelix thinks the above not worth further investigation
+* however, Kobzol has pointed out that the secondary regressions are significant, and has identified the root cause as PR [#128104](https://github.com/rust-lang/rust/issues/128104)
+* we are in any case planning to revert the changes to dead code analysis (see PR [#128404](https://github.com/rust-lang/rust/issues/128404)) which should address those regressions.
+* marked as triaged.
+
+#### Untriaged Pull Requests
```
Comment (lqd) [inline triage/2024-07-30.md:253]: it seems you did triage a bunch if not all all of the "untriaged pull requests" below?

File: triage/2024-07-30.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+- [#125120 Rollup of 7 pull requests](https://github.com/rust-lang/rust/pull/125120)
+- [#125069 Extend SCC construction to enable extra functionality ](https://github.com/rust-lang/rust/pull/125069)
+- [#124905 Allow optimizing `u32::from::<char>`.](https://github.com/rust-lang/rust/pull/124905)
+- [#124700 Remove an unnecessary cast](https://github.com/rust-lang/rust/pull/124700)
+- [#124417 Make early lints translatable](https://github.com/rust-lang/rust/pull/124417)
+- [#124241 Rollup of 5 pull requests](https://github.com/rust-lang/rust/pull/124241)
+
+#### Nags requiring follow up
+
+TODO: Nags
```
Comment (lqd) [inline triage/2024-07-30.md:288]: maybe we can remove this as well?

File: triage/2024-07-30.md (hunk 1 above)
Comment (pnkfelix) [reply triage/2024-07-30.md]: uh, maybe you meant to say "didn't triage"? I definitely didn't look at that list at all; I'm late enough delivering this report as it is...

File: triage/2024-07-30.md (hunk 2 above)
Comment (pnkfelix) [reply triage/2024-07-30.md]: yeah I did so in follow-up commit c12b75fb04eb51cb5f0a6eaae2398e2425aafbc7

**Review by lqd — APPROVED** [review]

**Pull request merged**